### PR TITLE
calendar-widget

### PR DESCRIPTION
**Pull request by nataliemisasi**
adding support page on calendar widget

### CODE REVIEW

File: help/widget-calendar.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+The **Calendar** widget allows viewing data in a calendar view. In Grist, event data is stored in a data table. Then, the **Calendar** widget can be configured to display that data.
+
+*![widget-calendar-intro](images/widget-calendar/widget-calendar-intro.png)*
+
+## Setting up your data
+
+In your data table, you will need three columns of data;
+
+1. [Text](col-types.md#text-columns) column, containing the title of your event.
+2. [DateTime](col-types.md#datetime-columns) column that will contain the date and time that the event begins.
```
Comment (nbush) [inline help/widget-calendar.md:12]: I think the fix for Date & DateTime support is in (https://gristlabs.getgrist.com/doc/check-ins/p/4#a1.s6.r2769.c19), so this should probably be updated to reflect that.

File: help/widget-calendar.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Page widget: Calendar
+
+The **Calendar** widget allows viewing data in a calendar view. In Grist, event data is stored in a data table. Then, the **Calendar** widget can be configured to display that data.
```
Comment (nbush) [inline help/widget-calendar.md:3]: The current version has several different stylings for "Calendar widget". I think lowercase "calendar widget" makes the most sense, but looking through the other related articles there's a bit of inconsistency (bolded + capitalized in intro paragraph; occasionally capitalized elsewhere).

Similar issue with "Card widget" later on in this article.

File: help/widget-calendar.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+*![widget-calendar-delete-record](images/widget-calendar/widget-calendar-delete-record.png)*
+
+If you do not have a linked widget, you can delete an event by clicking the three-dot icon at the upper-right of the Calendar widget then ‘Show raw data’.
+
+*![widget-calendar-raw-data](images/widget-calendar/widget-calendar-raw-data.png)*
+
+Find the event you want to delete, then delete that record.
+
+!!! note "Keyboard shortcut"
```
Comment (nbush) [inline help/widget-calendar.md:81]: I'd probably move this after the image.

**Review by nbush — CHANGES_REQUESTED** [review]
Looks great! A couple tiny change requests.

**Review by nbush — APPROVED** [review]

**Review by anaisconce — APPROVED** [review]

**Pull request merged**